### PR TITLE
update docs for auth flow usage 4.x

### PR DESCRIPTION
**Pull request by eriveltonelias**
I've created the `auth-flow.js` on example/4.x and replaced the snack link to in-line snack

### CODE REVIEW

File: website/versioned_docs/version-4.x/auth-flow.md [hunk 1]
```diff
@@ -84,7 +83,7 @@ class AuthLoadingScreen extends React.Component {
 }
 ```
 
-<a href="https://snack.expo.io/@react-navigation/auth-flow-v3" target="blank" class="run-code-button">&rarr; Run this code</a>
+<samp id="auth-flow">Auth Flow</samp>
```
Comment (WoLewicki) [inline website/versioned_docs/version-4.x/auth-flow.md]:
```suggestion
```

I would leave the snack only for the last example. @satya164 what you think?

Comment (satya164) [reply website/versioned_docs/version-4.x/auth-flow.md]: Sounds good 👍

Comment (eriveltonelias) [reply website/versioned_docs/version-4.x/auth-flow.md]: Done, can you check again @WoLewicki  ?

Comment (WoLewicki) [reply website/versioned_docs/version-4.x/auth-flow.md]: It's ok now. Thanks for your contribution!

**Pull request merged**